patches/components-policy-tools-template_writers-writer_configuration.py.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/components/policy/tools/template_writers/writer_configuration.py b/components/policy/tools/template_writers/writer_configuration.py
+index 259e467fee3e9280c512db87ab8f48ed09c20ed3..db6877f93cd8a216e39cc4f3f044dc4bba119912 100755
+--- a/components/policy/tools/template_writers/writer_configuration.py
++++ b/components/policy/tools/template_writers/writer_configuration.py
+@@ -131,3 +131,5 @@ def GetConfigurationForBuild(defines):
+     config['mac_bundle_id'] = defines['mac_bundle_id']
+   config['android_webview_restriction_prefix'] = 'com.android.browser:'
+   return config
++
```

Comment:
don't add empty lines pls.

Reply:
Resolved in 43bf0e0276e8c4291dc0b380643f1d4d090df11d.